modules/deployment-scripts/create-kv-certificate/main.bicep

```diff
@@ -86,65 +106,19 @@ resource createImportCert 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
     timeout: 'PT10M'
     retentionInterval: 'P1D'
     environmentVariables: [
-      {
-        name: 'akvName'
-        value: akvName
-      }
-      {
-        name: 'certName'
-        value: certificateName
-      }
-      {
-        name: 'certCommonName'
-        value: certificateCommonName
-      }
-      {
-        name: 'initialDelay'
-        value: initialScriptDelay
-      }
-      {
-        name: 'retryMax'
-        value: '10'
-      }
-      {
-        name: 'retrySleep'
-        value: '5s'
-      }
+      { name: 'akvName', value: akvName }
+      { name: 'certName', value: certificateName }
+      { name: 'certCommonName', value: certificateCommonName }
+      { name: 'initialDelay', value: initialScriptDelay }
+      { name: 'issuerName', value: issuerName }
+      { name: 'issuerProvider', value: issuerProvider }
+      { name: 'retryMax', value: '10' }
+      { name: 'retrySleep', value: '5s' }
+      { name: 'accountId', value: accountId }
+      { name: 'digicertPassword', value: digicertPassword }
```

Comment:
Use secureValue and not value for passwords
ref: https://learn.microsoft.com/en-us/azure/templates/microsoft.resources/deploymentscripts?pivots=deployment-language-bicep#environmentvariable

Reply:
```suggestion
      { name: 'digicertPassword', secureValue: digicertPassword }
```